### DISCUSSION TITLE
ImportError: cannot import name 'API' from 'chromadb.api' (/usr/local/lib/python3.10/dist-packages/chromadb/api/__init__.py)

### DISCUSSION THREAD

**Discussion by techybolek** [microsoft/autogen#449] [Q&A]
I ws a ble to run the RetrieveChat example in colab just a few days ago. As of today howerver I'm getting an error  atthe beginning when executing the following:

`from autogen.retrieve_utils import TEXT_FORMATS`

Error:
`ImportError: cannot import name 'API' from 'chromadb.api' (/usr/local/lib/python3.10/dist-packages/chromadb/api/__init__.py)`

**Comment by thinkall** [comment]
Hi @techybolek , this is fixed and will be released in the next version of pyautogen. Before that, could you manually install chromadb 0.4.14 . Thanks.
